docs/running_clusterfuzzlite.md

```diff
@@ -205,3 +161,4 @@ Follow the [subguide](#subguides) for your CI system to get started.
 [prune]: #prune
 [coverage]: #coverage
 [though this can be changed]: #configuration-options
+[sanitizers]: #sanitizers
```

Comment:
Wrong link: should go to `https://google.github.io/clusterfuzzlite/overview/#sanitizers`

Reply:
Changed to [sanitizers]: {{ site.baseurl }}/overview/#sanitizers (hoping that's right)

Reply:
I think that's right.